src/Uno.UI.RuntimeTests/Tests/Windows_UI_Composition/Given_CompositionClip.cs

```diff
@@ -1,366 +0,0 @@
-﻿using System;
```

Comment:
Why do you remove all those tests? Do they have to be udpated instead?

Reply:
They have been flaky for a while now. I rewrote them as UI samples to be checked by screenshot comparisons